forte/data/multi_pack.py

```diff
@@ -168,7 +174,32 @@ def _validate(self, entry: EntryType) -> bool:
 
     # TODO: get_subentry maybe useless
     def get_subentry(self, pack_idx: int, entry_id: int):
-        return self.get_pack_at(pack_idx).get_entry(entry_id)
+        r"""
+        This method now use pack_idx as pack_id to get a pack from multi pack,
```

Comment:
This docstring may still be improved, need to explain better what are `pack_idx` and `pack_id`